README.md

```diff
@@ -23,15 +23,10 @@ Installing all dependencies
 ## Run Tests
 - Run UI test
```

Comment:
We don't need to specify that this is UI test anymore since we are combining all the tests in one command.